plan/sort.go

```diff
@@ -78,9 +78,17 @@ func Sort(p *blip.Plan) []SortedLevel {
 					higherDomain = blip.Domain{
 						Name:    domain,
 						Metrics: []string{},
+						Options: map[string]string{},
+						Errors:  map[string]string{},
 					}
 				}
 				higherDomain.Metrics = append(higherDomain.Metrics, lower.Collect[domain].Metrics...)
+				for k, v := range lower.Collect[domain].Options {
+					higherDomain.Options[k] = v
```

Comment:
This is correct in general, but I think we also need to _not_ overwrite options set in higher domain. So a pathological case would be:

```
level_1:
  freq: 5s
  collect:
    foo:
      options:
        x: aaa
level_2:
  freq: 10s
  collect:
    foo:
      options:
        x: bbb
```

Option `x=aaa` should not overwrite explicitly set option `x=bbb` in the higher level. If I'm thinking clearly about this and you agree, let's also write a new unit test case for this.

Reply:
Talked offline. It makes sense to pick the options and errors of higher level.